docker/local-metadata/docker-compose.yml

```diff
@@ -30,6 +25,8 @@ services:
       - 3306
     ports:
       - "3306:3306"
+    volumes:
+     - $PWD/docker-volume/db-data:/var/lib/mysql
```

Comment:
Can you verify if the Github Actions CI dependent on this compose file will work as expected ?